.github/actions/db-backup/action.yml

```diff
@@ -0,0 +1,27 @@
+name: db-backup
+description: Backup postgres database to file
+inputs:
+  pass:
+    required: true
+  host:
+    required: true
+  port:
+    required: true
+  user:
+    required: true
+  database:
+    required: true
+  file-path:
+    required: true
+  options:
+    description: 'Additional pg_dump options'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - run: |
+        sudo PGPASSWORD=${{ inputs.pass }} \
```

Comment:
do you need a `sudo` here?

Reply:
It's because of the mount permissions. I could possibly modify the mount to work with non-root, but right now the mount is just using the example config from the docs.

Reply:
Makes sense, thanks!